.github/workflows/build-test-verilator.yml

```diff
@@ -34,96 +34,95 @@ jobs:
-            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
-            core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
-
-      - name: Restore verilator dir
-        uses: actions/cache/restore@v3
-        id: verilator_restore
-        with:
-          path: /opt/verilator
-          key: verilator-${{ env.VERILATOR_VERSION }}-${{ env.SCCACHE_C_CUSTOM_CACHE_BUSTER }}
-
-      - name: Install verilator
-        if: steps.verilator_restore.outputs.cache-hit != 'true'
-        run: |
-          sudo apt-get update -qy && sudo apt-get install flex bison libfl2 libfl-dev help2man
-          cd /tmp/
-          git clone -b "${VERILATOR_VERSION}" https://github.com/verilator/verilator
-          cd verilator
-          autoconf
-          ./configure --prefix=/opt/verilator CXX="sccache g++"
-          make -j6
-          sudo make install
-
-      - name: Save verilator dir
-        uses: actions/cache/save@v3
-        if: steps.verilator_restore.outputs.cache-hit != 'true'
-        with:
-          path: /opt/verilator
-          key: verilator-${{ env.VERILATOR_VERSION }}-${{ env.SCCACHE_C_CUSTOM_CACHE_BUSTER }}
-
-      - name: Setup verilator path
-        run: |
-          echo /opt/verilator/bin >> $GITHUB_PATH
-
-      - name: Update Cargo index
-        run: |
-          cargo tree --locked > /dev/null || (
-            echo "Please include required changes to Cargo.lock in your pull request"
-            exit 1
-          )
-
-      - name: Check that generated register code matches caliptra-rtl submodule
-        run: |
-          cargo run --locked -p caliptra_registers_generator -- --check hw/latest/rtl registers/bin/extra-rdl  hw/latest/registers/src
-
-      - name: Check that generated X.509 templates match default templates
-        run: |
-          cargo test -p caliptra-x509 --features=generate_templates
-
-      - name: Build
-        run: |
-          export RUSTC_WRAPPER=~/.cargo/bin/sccache
-          export CXX="sccache g++"
-          sccache --show-stats
-          (cd hw-model && cargo build --locked --release --features verilator --jobs 10)
-          # build all tests; need to make sure they still build with verilator
-          # even if we don't have time to run them.
-          cargo --config "$EXTRA_CARGO_CONFIG" test --no-run --locked --release --features=verilator
-          sccache --show-stats
-
-      - name: Run unit tests
-        run: |
-          export RUSTC_WRAPPER=~/.cargo/bin/sccache
-          export CXX="sccache g++"
-          (cd hw-model && cargo --config "$EXTRA_CARGO_CONFIG" test --locked --release --features verilator)
-          (cd hw/verilated && cargo --config "$EXTRA_CARGO_CONFIG" test --locked --release --features verilator)
-
-      - name: Check source-code formatting (run "cargo fmt" if this fails)
-        run: |
-          (cd hw-model && cargo fmt --check)
-
+      # - name: Restore sccache binary
```

Comment:
Why is this removed? If you're having an issue with the cache being incorrect you can change the cache buster value: https://github.com/chipsalliance/caliptra-sw/blob/470e9f9844d8d3c52149592986bf367ad08b44ea/.github/workflows/build-test-verilator.yml#L26

Reply:
So the whole verilator flow is not working on the 2.0 RTL and needs some work which I was told will be taken care of in a few weeks. I move forward commenting out most of this action was done. There isn't anything wrong with sscache afaik.

Reply:
Got it, thanks for clarifying